src/openrct2-ui/windows/EditorInventionsList.cpp

```diff
@@ -271,8 +271,9 @@ static ResearchItem* get_research_item_at(int32_t x, int32_t y, int32_t* outScro
         if (widgetIndex == WIDX_PRE_RESEARCHED_SCROLL || widgetIndex == WIDX_RESEARCH_ORDER_SCROLL)
         {
             gPressedWidget.widget_index = widgetIndex;
-            int32_t outX, outY, outScrollArea;
-            widget_scroll_get_part(w, widget, x, y, &outX, &outY, &outScrollArea, outScrollId);
+            int32_t outScrollArea;
+            ScreenCoordsXY outScrollCoords;
```

Comment:
Fun fact, these out coordinates are not used in this method

Reply:
I wander if `scrollY` is just `outScrollCoords.y + 6`

Reply:
It looks like `outScrollCoords.y + 4`, no?

Reply:
You have the sign of the 1 wrong. So its `-1 -> 5` which would be 6 different. I think it would be worthwhile changing it in this PR to not recalculate it cause its not like anyone will realise again for a long time.

Reply:
Changed it, it's too little to create another PR.

At least these refactorings are exposing some simplifications to us :D

Reply:
Yes thats one of the main ways of improving the codebase as many of the files go years without being reviewed.